src/extensions/ExtensionManager.as

```diff
@@ -162,15 +170,11 @@ public class ExtensionManager {
 		var ext:ScratchExtension = extensionDict[extensionName];
 		if (ext == null) return; // unknown extension
 
-		var index:int = ext.busy.indexOf(id);
```

Comment:
I'm nervous about removing the use of `ext.busy` for reporters / requesters. It looks OK at first glance but if possible I'd prefer a more conservative change that only implements HTTP requesters without affecting the way we track requesters in general.